packages/hoppscotch-app/assets/icons/arrow-down-left.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#eaaa45" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
```

Comment:
Icons should always have `currentColor`. Do not give fixed colors, instead use a computed class name to color the icons where they're being used.